apps/worker/src/app/workflow/services/metric-queue-active.service.ts

```diff
@@ -1,30 +1,14 @@
-const nr = require('newrelic');
 import { WorkerOptions } from 'bullmq';
 import { Inject, Injectable, Logger } from '@nestjs/common';
 import { QueueService } from '@novu/application-generic';
-import { min, max, sum, mean } from 'simple-statistics';
-
-interface IMetric {
-  count: number;
-  total: number;
-  min: number;
-  max: number;
-  sumOfSquares: number;
-}
-
-function sumOfSquares(values: number[]): number {
-  const meanVal = mean(values);
-
-  return sum(values.map((value) => (value - meanVal) ** 2));
-}
 
 const LOG_CONTEXT = 'MetricQueueService';
 const METRIC_JOB_ID = 'metric-job';
 
 @Injectable()
-export class MetricQueueService extends QueueService<Record<string, never>> {
+export class MetricQueueActiveService extends QueueService<Record<string, never>> {
   constructor(@Inject('BULLMQ_LIST') private token_list: QueueService[]) {
-    super('metric');
+    super('metric-active');
```

Comment:
What about adding the 'job' context as well??
because maybe in the future we will add other `metric-{SOMETHING}-active`
same for the other statuses.

```suggestion
    super(`${METRIC_JOB_ID}-active`);
```

Reply:
I dont think we should do this, this job is specificly for getting the active metric. 
I think if we need to add something to another job down the road we can do it then.